public/assets/js/card-flip.js

```diff
@@ -0,0 +1,47 @@
+let textDiv = document.getElementById("flip-text"); //catches the text
+let cardDiv = document.getElementById("card-flip"); //catches the card-flip ID
+let textDivContent = textDiv.innerHTML;
+console.log(textDivContent);
```

Comment:
I assume console log is for debugging, you can remove'em before to push it on dev branch